incident-bot.json

```diff
@@ -341,17 +449,21 @@
    },
    "WaitCondition" : {
      "Type" : "AWS::CloudFormation::WaitCondition",
-     "DependsOn" : "IncidentBotInstance",
+     "DependsOn" : "BotInstance",
      "Properties" : {
          "Handle" : { "Ref" : "BotWaitHandle" },
          "Timeout" : "900"
       }
     }
   },
   "Outputs": {
```

Comment:
What is the Outputs section do?  Trying to understand where this fits in, but do not see it referenced anywhere.